README.md

```diff
@@ -11,7 +11,7 @@ The Makefile is self-documented. It means, that by running `make` it will print
 
 ```shell
 ❯ make
-build      Build a docker image tagged mysql-devenv
+build      Build a docker image tagged mysql-devenv (default: Debian)
```

Comment:
This is auto-generated from the Makefile target's comment, therefore, I suggest you to add `(default: Debian)`  [here](https://github.com/malderete/mysql-devenv/blob/main/Makefile#L9)